app/controllers/checklist_templates_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ChecklistTemplatesController < InheritedResources::Base
+
+  private
+
+    def checklist_template_params
+      params.require(:checklist_template).permit(:name, :string, :description, :text)
```

Comment:
I don't think `.permit` works this way, ie: you only need to list the field names (`:name` and `:description`) and not the types (`:string` and `:text`).